iree/turbine/kernel/wave/expansion.py

```diff
@@ -566,53 +613,63 @@ def _handle_reduction_dim(
     # TODO: Register iter args with the reduction initially so accessing them is easier
     iter_args: list[CustomOp] = []
     reduction_subgraph = trace.get_subgraph(reduction.subgraph_name)
+
+    # TODO: Add support for case where we process MMA before returning to IterArg.
```

Comment:
Don't fully follow this comment, can you explain?

Reply:
Cleared up the comment to "Handle case where MMAs/ReduceOps do not have Output as direct consumer."
LMK if it's still not well worded :)

Reply:
yes that makes a lot more sense, thanks